tedana/workflows/parser_utils.py

```diff
@@ -12,3 +13,17 @@ def is_valid_file(parser, arg):
         parser.error('The file {0} does not exist!'.format(arg))
 
     return arg
+
+
+class ContextFilter(logging.Filter):
+    """
+    This is a filter which injects contextual information into the log.
+
+    Rather than use actual contextual information, we just use random
+    data in this demo.
```

Comment:
What's this demo included for? And is there any reason it's a class instead of a function?